### DISCUSSION TITLE
Nominatim UI

### DISCUSSION THREAD

**Discussion by gsg-git** [mediagis/nominatim-docker#486] [Q&A] [answered]
I have my container running, but i have to set up the UI.

Can i just normal continue with the guide  and the UI will be reachable via IP:8080/ui 
or do i have to set up another Docker port for the Apace2 bevore i continue?

Thank you!

**Comment by leonardehrenfried** [comment]
The UI is not included in the image and you will have to install it separately: https://github.com/osm-search/nominatim-ui

**Reply by gsg-git** [reply]
I know, but i have the system running and can search with http://192.168.19.200:8080/search.php?q=africa.
but somehow the API doesnt seem to be reachable for the UI even though inconfigiured everything like in the guide.

i dont know if i have to use for the cofig the defauilt 8080 Port in the amache conf and API URL. because everything is on the same client.
or do i have to make in docker an own Port for the Apache website  - i dont really know anymore.
Thank you!

**Reply by mtmail** [reply]
Please @gsg-git You opened three issues on three repositories today (three different groups of people), all about the same issue. Volunteers gets exhausted by the amount of questions. Let us answer one question at a time, nobody works on Nominatim 24/7 and response time of several days is normal.

**Reply by gsg-git** [reply, marked as answer]
@leonardehrenfried , @mtmail 

Hello!
I found the source of the problem:
You have to set the permissions for the user/group www-data at /nominatim/website to 755.
In the regular Nominatim [Handbook](https://nominatim.org/release-docs/latest/appendix/Install-on-Ubuntu-22/) is the step where the permissions at the beginning. 
Where someone propably would not look for, when they use the Docker Nominatim.
I think maybe it would be better if the Nominatim Docker Setup maybe sets the Permissions?
I dont know how much efford this will take or if its a security risk to have the permissions set per default?
Or should i add it to my PR-in-work for Docker Nominatim Doku or make a new at Nominatim UI?

Thank you!

**Reply by mtmail** [reply]
Thanks for feedback, I haven't thought about it. So the error would've shown up in the apache error.log.

Both Nominatim and Nominatim-Docker project treat the nominatim-ui as separate add-on that only few users need. I'll add a section in nominatim-ui README.md to point out how to install it with docker in the next couple of days. I've had the request to explain a docker-compose setup, too.

**Reply by gsg-git** [reply]
@mtmail Yeah the log  was 403 forbidden.  "search permissions are missing on a component of the path" 
For me the first thought was: Eh? is there really a permission type i didnt got in all these years!?
I added for the /website to read and execute recursively, but it wasnt enoght and i found in the web mostly 775 in the guides.
[This thread helped](https://askubuntu.com/questions/451922/apache-access-denied-because-search-permissions-are-missing)
